Get-ADGroupsDifference.ps1

```diff
@@ -103,151 +125,117 @@ Function Get-ADGroupsDifference {
         [String]$DomainName,
         [parameter(Mandatory = $false)]
         [Switch]$IncludeEqual
-        
     )
-    
-    BEGIN {
-        
         
-        if ($null -eq (Get-Module -name 'ActiveDirectory' -ErrorAction SilentlyContinue)) {
-            
+    BEGIN {
+
+        if ($null -eq (Get-Module -name 'ActiveDirectory' -ErrorAction SilentlyContinue)) { 
             Import-Module -Name 'ActiveDirectory' -ErrorAction Stop | Out-Null
-            
         }
-        
-        If ($DomainName -eq $Null) {
             
+        If ($DomainName -eq $Null) {
             $DomainName = (Get-ADdomain -Current LoggedOnUser).DNSRoot
-            
         }
-        
+            
         [String]$DomainController = (Get-ADDomainController -DomainName $DomainName -Discover).HostName
-        
+            
         $Results = @()
-        
+
+        function ConvertFrom-DN {
+            #Based on: http://practical-admin.com/blog/convert-dn-to-canoincal-and-back/
+            #Credit: Andrew
+            #Corrected by Wojciech Sciesinski 
+                
+            param ([string]$DN = (Throw '$DN is required!'))
+                
+            foreach ($item in ($DN.replace('\,', '~').split(","))) {
+                switch -regex ($item.TrimStart().Substring(0, 3)) {
+                    "CN=" { $CN += , $item.replace("CN=", ""); $CN += '/'; continue }
+                    "OU=" { $OU += , $item.replace("OU=", ""); $OU += '/'; continue }
+                    "DC=" { $DC += $item.replace("DC=", ""); $DC += '.'; continue }
+                }
+            }
+                
+            $canonical = $dc.Substring(0, $dc.length - 1)
+                
+            If ($ou.count -gt 0) {
+                for ($i = $ou.count; $i -ge 0; $i--) { $canonical += $ou[$i] }
+            }
+                
+            If ($CN.count -gt 0) {
+                for ($i = $CN.count; $i -ge 0; $i--) { $canonical += $CN[$i] }
+            }
+                
+            return $canonical
+        }
+
+        # Define custom default view
+        $defaultDisplaySet = 'ReferenceUser', 'User', 'GroupName', 'SideIndicator', 'SideIndicatorName'
+
+        # Create the default property display set
+        $defaultDisplayPropertySet = New-Object System.Management.Automation.PSPropertySet(‘DefaultDisplayPropertySet’, [string[]]$defaultDisplaySet)

```

Comment:
In this line are used some strange "quotation" chars - you probably copied it from a website.